### DISCUSSION TITLE
Show more weather details on the main screen.

### DISCUSSION THREAD

**Discussion by EdgarOns** [breezy-weather/breezy-weather#1340] [Ideas]
### Describe your idea

I think the main screen of the app doesn't show enough data, since you have to swipe down to see conditions like barometric pressure, UV index, visibility, etc. Additionally, wind gusts are not displayed in the current conditions details. To see it you have to click on the hourly forecast, which makes it difficult to view and more complex to quickly interpret the metrological data. The largest amount of meteorological data or parameters should appear on the main screen so that users can make a quick and intuitive reading.

Creo que en la pantalla principal de la aplicación no se muestran suficientes datos, ya que hay que deslizar hacia abajo para ver condiciones como la presión barométrica, índice UV, visibilidad, etc.
Además, no se muestran las rachas de viento en los detalles de las condiciones actuales. Para verlo tienes que hacer click en el pronóstico por horas, lo cual dificulta su visualización y la hace más compleja para interpretar rápidamente los datos metrológicos.
En la pantalla principal deben aparecer la mayor cantidad de datos o parámetros meteorológicos a fin de que los usuarios puedan hacer una lectura rápida e intuitiva.

### Acknowledgements

- [X] I have searched the [existing issues](https://github.com/breezy-weather/breezy-weather/issues) and [existing discussions](https://github.com/breezy-weather/breezy-weather/discussions/categories/ideas) and this is a new ticket, **NOT** a duplicate or related to another open OR closed issue or discussion.
- [X] The idea I’m suggesting is not in the [list of features that will not be implemented](https://github.com/breezy-weather/breezy-weather/blob/main/README.md#features-that-will-not-be-implemented).
- [X] I have updated the app to **[version 5.2.8 or later](https://github.com/breezy-weather/breezy-weather/releases/latest)**. Even if I’m getting updates from F-Droid default repo, I don’t need to uninstall/reinstall to update.
- [X] I wrote my request in English, and any screenshot I attached are also in English (language can be changed from Settings > Appearance). I can write in my native language under the English text, if necessary.
- [X] I understand that this app is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY, so I may not get a positive reply to my request, if any at all.

**Comment by papjul** [comment]
Pressure, visibility and UV index are visible from the main screen. By default, it will be visible if your source has support of it (they may have been disabled in Main screen settings if you don't see them).

Regarding wind gust, it's a duplicate of #22
